pkg/scheduler/scheduler_test.go

```diff
@@ -1181,15 +1175,11 @@ func TestSchedulerBinding(t *testing.T) {
 			stop := make(chan struct{})
 			defer close(stop)
 			scache := internalcache.New(100*time.Millisecond, stop)
-			algo := NewGenericScheduler(
-				scache,
-				nil,
-				0,
-			)
-			sched := Scheduler{
-				Algorithm:      algo,
-				Extenders:      test.extenders,
-				SchedulerCache: scache,
+			sched := &Scheduler{
+				Extenders:                test.extenders,
+				SchedulerCache:           scache,
```

Comment:
use SchedulerCache: internalcache.New(100*time.Millisecond, stop) ？
Then we can delete L1177

Reply:
Thanks for your review!
I have changed that one.